values/minio-values.yaml

```diff
@@ -10,7 +10,13 @@ persistence:
   size: 50Gi
 
 buckets:
-  - name: test
+  - name: uds-loki
```

Comment:
Can we keep this generic so that it's more usable across other potential uses? Either generic `uds` or `dev` is fine with me - just don't like putting specific names like Loki in here. Does that still work for this need?

Reply:
yea, that should work.  Updated to be `uds` user and bucket